js/whitebit.js

```diff
@@ -95,6 +100,55 @@ module.exports = class whitebit extends Exchange {
                         'trades/{market}',
                     ],
                 },
+                'publicV4': {
+                    'get': [
+                        'assets',
+                        'ticker',
+                        'trades/{market}',
+                        'fee',
+                        'assets',
+                        'time',
+                        'ping',
+                    ],
+                },
+                'privateV1': {
+                    'post': [
```

Comment:
Although we're using the V4 I added it here the V1 also just to make it available if the users want